Tactics/SymContext.lean

```diff
@@ -0,0 +1,230 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Alex Keizer
+-/
+import Lean
+import Arm.Exec
+
+/-!
+This files defines the `SymContext` structure,
+which is collects
```

Comment:
```suggestion
This files defines the `SymContext` structure,
which tracks the various theorems needed for evalulation. This simplifies implementing proof automation, since automation can look for `SymContext` in the hypothesis, rather than hardcoded theorem names.
```

Reply:
This is incorrect: `SymContext` tracks `Name`s at meta-time, it would never show up in the local context